model/change_point_detection.go

```diff
@@ -312,7 +325,7 @@ func GetPerformanceData(ctx context.Context, env cedar.Environment, performanceR
 func ReplaceChangePoints(ctx context.Context, env cedar.Environment, performanceData *PerformanceData, mappedChangePoints map[string][]ChangePoint) error {
 	err := clearChangePoints(ctx, env, performanceData.PerformanceResultId)
 	if err != nil {
-		return errors.Wrapf(err, "Unable to clear change points for measurement %s", performanceData.PerformanceResultId)
+		return errors.Wrapf(err, "Unable to clear change points for measurement %s", performanceData.PerformanceResultId.Project)
```

Comment:
Why?

Reply:
I was getting complaints from the linter. Was a warning rather than an error. I can change them back if this was intended

Reply:
I changed it back to how it was